templates/config/config.cue

```diff
@@ -75,9 +69,25 @@ import (
 		retryPeriod?: #Duration
 	}
 
-	controller: #Controller
-	webhook:    #Webhook
-	caInjector: #CAInjector
+	controller: #Controller & {
+		replicas: *1 | uint16 & >1
+		if highAvailability.enabled {
+			replicas: *2 | uint16 & >2
+		}
+	}
+	webhook: #Webhook & {
+		replicas: *1 | uint16 & >1
+		if highAvailability.enabled {
+			replicas: *3 | uint16 & >3
+		}
+	}
+	caInjector: #CAInjector & {
+		replicas: *1 | uint16 & >1
+		if highAvailability.enabled {
+			replicas: *2 | uint16 & >2
+		}
+	}
+
```

Comment:
Would it make more sense to have the `replicas: *1 | uint16 & >1` be in the original definition rather than here?

Reply:
Fixed